conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
So, Anaconda charges for the NumPy version that is built with MKL. I think this is largely due to the licensing fees of MKL. In other words, the version of NumPy linked to MKL is not commonly provided to users. If you want to make sure that MKL is linked about when the user has access, we can discuss this.

The normal version of NumPy for Linux links against OpenBLAS. As they do provide this, it would be no problem to add it to this recipe. However, they only provide this for Linux. On Mac, they link against Accelerate. As I believe you well know, Accelerate is not fork safe. Given the prevalence of forking in Python code, this seems like a bad idea, but they haven't changed it. On Windows, it appears they are not using a BLAS.

My understanding, based on the related issue, was to provide a way to make development builds not release builds. Adding a conda recipe would help facilitate people already using conda to build and or try these development versions. As a result, I don't think we need to stress too much about the optimal builds of NumPy or how to facilitate normal releases with these.

On the issue of visibility, [anaconda.org](https://anaconda.org/) provides the ability to create a user and/or organization and you can upload binaries there. These can be easily downloaded within `conda`. Also, they provide badges if we wanted to put one in the README or something.

As for the `conda` NumPy ABI, there is nothing special that needs to be done to the recipe to get it to track the NumPy version. I build my own versions of NumPy linked to OpenBLAS and have yet to need to do anything special on that front. Packages built with a NumPy dependency on the other hand are normally made dependent on a particular NumPy major and minor version and should normally be rebuilt. However, one can always tell `conda` to force install a version of NumPy into an environment with packages not tied to its version, but this shouldn't be recommended.